### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by ryebytescode** [revel/revel#1575]
Is this project still maintained?

**Comment by fullgukbap** [comment]
I, too, would like to see a moderator's answer to this issue.

**Comment by alnwt** [comment]
Hi, @brendensoares.

*TLDR: There are a number of things I'm going to work on, would you be willing to have them under the umbrella of @revel project, probably to be released later as a backward incompatible Revel v.2?*

I'm about to start working on implementation of a framework that could provide a complete full-stack experience requiring as little boilerplate as possible. While still using the approach of Revel controllers and code generation I expect it to produce not only back-end code but also production-grade web and mobile clients. That is, the ones that could be competitive with existing JS solutions. To be more concrete, some of the ideas I have in mind:

1. Structure the project the way proposed earlier and followed by some Revel offshoots/rewrites: a set of independent tools at the center (revel/revel), user app is a regular standard library compatible program but with Revel style magic controllers invocation; other packages, being easily replaceable in the user's app, could live in their own repositories (revel/*).
2. Use something akin to [elm-ui](https://package.elm-lang.org/packages/mdgriffith/elm-ui/latest/) to abstract HTML & CSS away leaving only common building blocks for all platforms.
3. Use either [LiveView Paradigm](https://www.liveviewjs.com/docs/overview/paradigm) style or [htmx](https://htmx.org/) like approach, or consider some form of code generation that would produce assets of reasonable size and speed.

Also to be addressed but no specific ideas yet:

1. Decide how Hot Code Reload could keep app's state between restarts.
2. Come up with a solution for models and automatic app deployment.

Is it something that can be done under the umbrella of @revel project? If so, I'll work on its fork so it can be merged back later. If not, I'll start from scratch.

**Comment by mydreamdoctor** [comment]
I cam to ask the same question. I've just started looking at Go as a language, and would love to use it for a simple web project I'm working on, however every framework I look at seems to have stopped development a year or two ago, which is very disappointing, as Go seems like such a fantastic language, especially for a web project. Has Rust taken over everyone's interest or something?

**Comment by ManuelReschke** [comment]
@baylf2000 take a look at [Go-Fiber](https://github.com/gofiber/fiber)

**Comment by bastianwegge** [comment]
@notzippy @brendensoares 

I would also be interested in this projects future plans, if there are any.

**Comment by ptman** [comment]
you can try to interact on gitter.im (revel/community), but it's quite dead

would make sense to archive

**Comment by brendensoares** [comment]
@bastianwegge, where are you located, what's your area of strength, what's your availability, what would you like to focus/lead on?

**Comment by brendensoares** [comment]
@nobrelli it is essentially not actively maintained, but I have had at least 2 people reach out wanting to volunteer. I'd love to get it going again, it's just been a lower priority for me personally.

**Comment by wiselike** [comment]
I forked the repository into [wiselike-revel](https://github.com/wiselike/revel) and fixed some issues both in [revel](https://github.com/wiselike/revel), [cmd](https://github.com/wiselike/revel-cmd), [config](https://github.com/wiselike/revel-config), [modules](https://github.com/wiselike/revel-modules), [log15](https://github.com/wiselike/revel-log15) .   
However, please kindly note that the forked repository has been deprecated too and is now only maintained for old Revel projects.  
I just want my old revel-project to keep running, and without vulnerability.
